src/components/transactions/message-view/ClaimRewardsTxDetails.vue

```diff
@@ -41,7 +41,11 @@
     >
       <div class="tx__content__left multi-claim-reward-row">
         <h3 class="multi-claim-reward-h3">{{ caption }}</h3>
-        <div class="multi-claim-reward-row" :class="{ validatorsToggle: show }">
+        <div
+          v-if="getValidators.length > 0"
```

Comment:
why is this necessary? line 50 handles an empty array gracefully

Reply:
You mean line 39? true... I didn't notice. So this is then no the fix probably.

OK, but please don't close this PR as this is still an issue. Will look into it

Reply:
OK, just fixed it the real way. But we really need to fix API taking too much time to load validators